docs/plans/go-ci/hello/go.mod

```diff
@@ -0,0 +1,3 @@
+module dagger.io/testgreetci
+
+go 1.17
```

Comment:
1.18?

Reply:
Well, it doesn't have too. The example doesn't use generics nor fuzzing or specific go 1.18 features. So go 1.17 is good.
I would even say that it could work with go 1.13.
I just copied it from the `./pkg/universe.dagger.io/go/`, didn't modify anything except the plan to make them graspable.